Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2618,27 +2146,20 @@ script:
       description: The incident score.
       type: Number
   - arguments:
-    - default: false
-      description: The endpoint ID.
-      isArray: false
+    - description: The endpoint ID.
       name: id
-      required: false
-      secret: false
     - default: true
       description: The endpoint IP address.
-      isArray: false
       name: ip
-      required: false
-      secret: false
-    - default: false
-      description: The endpoint hostname.
-      isArray: false
+    - description: The endpoint hostname.
       name: hostname
-      required: false
+    - default: true
+      description: ''
+      isArray: true
+      name: endpoint
+      required: true
```

Comment:
Can you please explain why did we add this argument?
I can't find any use for this param in .py file.

Reply:
Again, I didn't mean to make these changes as well. I am guessing these got added somehow when I made the contribution through the XSOAR UI while validating. Need to check more on that. But for now I have reverted the changes.